model/oracledb/registry.yaml

```diff
@@ -0,0 +1,42 @@
+groups:
+  - id: registry.oracledb
+    type: attribute_group
+    display_name: Oracle Database Attributes
+    brief: >
+      This section defines attributes for Oracle Database.
+    attributes:
+      - id: oracledb.implicit_release
+        type: boolean
+        stability: development
+        brief: >
+          Boolean flag Indicating if the internal connection is released back to pool or not after executing a query.
+        examples: [true]
+      - id: oracledb.instance
+        type: string
+        stability: development
+        brief: The name of the Oracle Database instance.
+        examples: "ORCL1"
+      - id: oracledb.pdb_name
+        type: string
+        stability: development
+        brief: >
+          The name of the pluggable database (PDB) the connection is using.
+        examples: "PDB1"
+      - id: oracledb.pool.increment
+        type: int
+        stability: development
+        brief: >
+          The number of connections added when the pool needs to expand.
+        examples: 5
+      - id: oracledb.pool.max
```

Comment:
please check out DB connection pool metrics - https://github.com/open-telemetry/semantic-conventions/blob/main/docs/database/database-metrics.md#connection-pools and check if anything oracle-specific is necessary.

Reply:
Thanks for the link. I removed `oracledb.pool.max` and `oracledb.pool.min` as they are specified in standard as
 `db.client.connection.max` and `db.client.connection.idle.min  `respectively.
 
I was adding these parameters in span when application creates a pool with the pool configuration (max, min ,..). I think the standard talks about this attribute to be included in metrics as `UpDownCounter`, Can I still keep the same name in spans .

Reply:
I removed oracledb.pool.max,  oracledb.pool.incr and oracledb.pool.min .